File: embrace/example/android/app/src/main/java/io/embrace/flutter/example/MyApplication.java
package io.embrace.flutter.example;

import android.app.Application;
import androidx.multidex.MultiDexApplication;
import io.embrace.android.embracesdk.Embrace;

public final class MyApplication extends MultiDexApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        Embrace.getInstance().start(this, false, Embrace.AppFramework.FLUTTER);
    }
}

File: embrace/example/android/app/src/main/kotlin/io/embrace/flutter/example/MyApplication.kt
package io.embrace.flutter.example

import android.app.Application
import androidx.multidex.MultiDexApplication
import io.embrace.android.embracesdk.Embrace

class MyApplication : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        Embrace.getInstance().start(this, Embrace.AppFramework.FLUTTER)
    }
}
